general/administration/hardware-acceleration.md

```diff
@@ -49,6 +49,117 @@ The hardware acceleration is available immediately for media playback. No server
+
+7. Now your should see a terminal login with your user
+          IE. my user is ```devilscoder```
+          so i loing with user ```devilscoder``` and the password associated with it.
+
+9. Now make the this directory and navigate to it by sending this command.
+        ```sudo mkdir /opt/nvidia && cd /opt/nvidia```
+
+10. Now we need to download the correctly update driver from the command posted below
+        ```sudo wget https://international.download.nvidia.com/XFree86/Linux-x86_64/(driver firmware number)/NVIDIA-Linux-x86_(driver firmware number).run```
+        IE if my drivers firmware revision is 335.21 that we found on the nvidia driver search on hte nvidia website. So mine would look like this
+        ```sudo wget https://international.download.nvidia.com/XFree86/Linux-x86_64/435.21/NVIDIA-Linux-x86_64-435.21.run```
+
+11. Now we need to give them permissions.
+        ```sudo chmod +x ./NVIDIA-Linux-x86_64-(driver firmware number).run```
+
+12.Now we need to run the driver installer.
+           ```./NVIDIA-Linux-x86_64-430.50.run```
+
+14. Make sure the new drivers is installed and recognized by typing in
+            ```nvidia-smi```
+
+15. Once the driver is installed we now need to install and patch the driver with the patch file downloaded from the github.
+            ```cd ~/```
+            ```sudo git clone https://ipfs.io/ipns/Qmed4r8yrBP162WK1ybd1DJWhLUi4t6mGuBoB9fLtjxR7u nvidia-patch```
+            ```bash ./patch.sh```
+16. Now it should display succesfully patched. Now that everythings done we can restart xServer or whatever you were using for DE (Desktop Enviroment)    
+            ```startx```
+            Your Desktop Enviroment should then display login and move your way to your jellyfin install in browser
+            ie. (internal ip adress of server):8096 ie. 192.168.1.155:8096
+            ie. (if your on the computer where jellyfin is installed)       http://localhost:8096 
+            ie. (if you have a reverse-proxy running) https://sub.yourdomain.com        https://stream.jellyfin.com
+17. Once there click on the top left hamburger menu and click on dashboard in the menu
+
+18. Once Loaded click on the hamburger menu again and select playback
+
+19. Now is IMPORTANT!!! Open this link [NVENC/DECODING SETTINGS FOR GPU](https://developer.nvidia.com/video-encode-decode-gpu-support-matrix#Encoder)
+                Once there if your card is 2 years or older it will most likely not have NVENC (NVIDIA ENCODING) on the card. So lets go  scroll to the bottom and select the green button that says Geoforce/Titan becuase mine in this example is a Geoforce GTX 970.
+        Now it should pull up more graphics card. Hit the key combinations CTRL + F and type your graphics card so im going to type 970. 
+        it will take you to your card and now check the settings mine has these decoding features. 
+            MPEG-1   YES      MPEG-2   YES      VC-1   YES      H.264(AVCHD)   YES
```

Comment:
H264 is required for jf transcoding to work

Reply:
ill add this to the docs